reactor-core/src/test/java/reactor/core/scheduler/SchedulersMetricsTest.java

```diff
@@ -3,36 +3,41 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 
-import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.Metrics;
 import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
+import reactor.core.Disposable;
+import reactor.core.Disposables;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class SchedulersMetricsTest {
 
 	final SimpleMeterRegistry simpleMeterRegistry = new SimpleMeterRegistry();
+	Disposable.Composite toCleanUp;
 
 	@Before
 	public void setUp() {
 		Metrics.addRegistry(simpleMeterRegistry);
 		Schedulers.enableMetrics();
+		toCleanUp = Disposables.composite();
```

Comment:
I'm curious why you didn't make it `final` and initialize in the field's declaration? JUnit creates a new instance of the test class for each test anyways